AboutFIBOProd.rdf

```diff
@@ -48,10 +48,30 @@
     <owl:imports rdf:resource="https://spec.edmcouncil.org/fibo/ontology/FND/AgentsAndPeople/Agents/"/>
     <owl:imports rdf:resource="https://spec.edmcouncil.org/fibo/ontology/SEC/Securities/SecuritiesIdentification/"/>
     <owl:imports rdf:resource="https://spec.edmcouncil.org/fibo/ontology/FBC/FunctionalEntities/BusinessCentersIndividuals/"/>
+<<<<<<< HEAD
```

Comment:
To remove

Reply:
@trypuz I'm finding it hard to understand what GitHub's telling me about what you want @ElisaKendall to change. Can you please describe the changes you want and why.
Note that the reason this has been held up (see comments from me and Mike) is that this is expanding into a totally new area that seems outside the scope of FIBO and not supporting any documented FIBO use case. And needs a more general presentation and architectural discussion which would have been with the FLT and now I guess the Steering Team (?) which has not been constituted yet.

Reply:
It's about line 51.

Reply:
The process for generating the AboutFIBOProd and AboutFIBODev ontologies appears to be broken.  I'll fix the conflict in this pull request, but at a higher level, the broader issue needs to be addressed as well.  Having an ontology IRI that is "file:///AboutFIBO" is wrong for so many reasons, and there is no metdata at all in these files.